TTPChallenge/ITEngVC.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ITEngVC.swift
+//  TTPChallenge
+//
+//  Created by Mirim An on 2/15/17.
+//  Copyright © 2017 TeamMDC. All rights reserved.
+//
+
+import UIKit
+
+class ITEngVC: UIViewController {
+    
+    @IBOutlet var informationButtons:[UIButton]!
+    
+    @IBOutlet weak var retakeQuizButton: UIButton!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        for button in informationButtons {
+            button.layer.cornerRadius = 12
+        }
+        retakeQuizButton.layer.cornerRadius = 20
```

Comment:
see above for how to change the corner radius in storyboard. let me know if you have any questions